src/js/components/AppListComponent.jsx

```diff
@@ -192,34 +192,34 @@ var AppListComponent = React.createClass({
   getGroupsFromApps: function (apps) {
```

Comment:
The list of items at this point includes apps (ex `/group/app`) as well as empty groups (ex. `/empty-group`).  Essentially the only difference here is that the `flattened` app array contains ALSO empty groups.